docs/docs/development/hardware-integration/new-shield.mdx

```diff
@@ -511,6 +511,19 @@ If all of your physical layouts use the same `kscan` node under the hood, you ca
 
 :::
 
+Additionally, if testing the experimental [wired split](../../features/split-keyboards.md) support, you should assign a [predefined](./pinctrl.mdx#predefined-nodes) or [pinctrl configured](./pinctrl.mdx) UART to the `zmk,split-uart` chosen property:
```

Comment:
Just leaving a comment here as a reminder to update this section.

Reply:
I added this in a new section, since it's no longer just an additional chosen node... But that means a placeholder text for unibody shields... Thoughts?